spec/helpers/hits_helper_spec.rb

```diff
@@ -92,25 +92,25 @@
     end
 
     context "when a site is present" do
-      let(:site) { build :site, abbr: "site_abbr" }
+      let(:site) { create :site, abbr: "site_abbr" }
```

Comment:
Can the `abbr` be removed now?

Reply:
Yes, that's a good idea, so I've done this in all cases where we're not using the `abbr` in a test.

It also revealed some other code that depended on `abbr` but wasn't caught by other tests, so I've updated those to use the ID too.